cockroachcloud/use-managed-service-backups.md

```diff
@@ -125,6 +124,23 @@ Additional ways to restore data:
 - [Back up a self-hosted CockroachDB cluster and restore into a {{ site.data.products.db }} cluster](#back-up-a-self-hosted-cockroachdb-cluster-and-restore-into-a-cockroachdb-cloud-cluster)
 - [Back up and restore data manually](take-and-restore-customer-owned-backups.html)
 
+### Restore a cluster
+
+To restore a cluster:
+
+1. Find the cluster backup on the **Backups** page.
```

Comment:
You'll know better than me, but the screenshot at the top of the Dedicated section... is that still correct? That is the Backups Page too. I know this is a different section, but I wonder if that needs updating? It might not, I have no idea. Is there a restore button that is actually on the side of that?

Reply:
Good catch. The Backups page screenshot is still correct but now has the 'Restore' button on the side. Updating the image to include this in the feature!